content/docs/react-without-jsx.md

```diff
@@ -37,12 +37,11 @@ ReactDOM.render(
   document.getElementById('root')
 );
 ```
+Próbáld ki [az online Babel fordítót](babel://jsx-simple-example), ha szeretnél több példát látni rá, hogyan alakítjuk át a JSXet JavaScriptté.
```

Comment:
It looks like there is a line misalignment from here downwards. Please fix that, so it looks exactly like the original (English) text.

Reply:
Sure!